docs/Manual/04 Fusing.md

```diff
@@ -116,55 +116,57 @@ for i in range(0, 16, 4):
+To understand this change in the logic, note that the resulting schedule is equivalent to the following Python code:
 ```python
 for a in range(16):
     for b in range(16):
         C[a, b] += A[a, b]
         C[b, a] *= B[b, a]
 ```
-In particular, this code sets `C[1,0]` to `C[1,0] * B[1,0] + A[1,0]`, whereas the original fused logic set `C[1,0]` to `(C[1,0] + A[1,0]) * B[1,0] `. We conclude that `schedule_t` is certainly not safe. If the programmer creates an unsafe schedule, they take upon themselves the responsibility of maintaining logical equivalence.
+The above code sets `C[1,0]` to `C[1,0] * B[1,0] + A[1,0]`, whereas the original fused logic set `C[1,0]` to `(C[1,0] + A[1,0]) * B[1,0] `. In this case, we can conclude that `schedule_t` is definitely not safe. If the programmer decides to create an unsafe schedule, Accera can no longer guarantee safety, and they take upon themselves the responsibility of maintaining logical equivalence.
 
 ### Fusing iteration spaces with different shapes
 If the iterations spaces have different shapes, Accera matches their shapes by padding them appropriately with empty cells.
 
 ## Partial fusing
-In many cases, instead of fusing all of the dimensions, we only need to fuse some of the dimensions, leaving the rest unfused. To fuse the first *s* dimensions, we use the syntax
+Instead of fusing all the dimensions, most of the time, we only need to fuse some of the dimensions, leaving the rest unfused. To fuse the first *s* dimensions, we use the syntax:
 ```python
 # Fuse the first s dimensions of three schedules
 schedule = acc.fuse((schedule0, schedule1, ...), partial=s)
 ```
-The order of the dimensions in the fused schedule is as follows: first the fusing dimension `f`, then the *s* fused dimensions, then the unfused dimensions of `schedule0`, `schedule1`, etc.
+The sequence of the dimensions in the fused schedule is as follows: first the fusing dimension `f`, then the fused dimensions *s*, followed by the unfused dimensions of `schedule0`, `schedule1`, and so on.
 
 We can easily calculate the number of dimensions in the fused schedule. For example, if we fuse the first *s* dimensions of a *d0*-dimensional space `schedule0` and a *d1*-dimensional space `schedule1`, the fused iteration space will have *s* fused dimensions, *d0 + d1 - 2s* unfused dimensions, and the special fusing dimension `f`, for a total of *d0 + d1 - s + 1* dimensions.
 
-As before, the `fuse` operation uses padding to ensure that the fused iteration space is not jagged in any direction. For example, say that `schedule0` is 4-dimensional, `schedule1` is 3-dimensional, and we partially fuse their first 2 dimensions:
+As established earlier, the `fuse` operation uses padding to ensure that the fused iteration space is not imbalanced in any direction. For example, consider that we partially fuse the first 2 dimensions of `schedule0`, which is 4-dimensional, and `schedule1`, which is 3-dimensional:
```

Comment:
"jagged" is correct because we are talking about the shape of the space.

("imbalanced" does not apply to iteration space shapes here)